src/TupleTypes.jl

```diff
@@ -2,15 +2,18 @@ module TupleTypes
 
 import Base: isvatuple
 
-function parameters{T<:Tuple}(::Type{T})
```

Comment:
The reason I had these as separate functions was because although the parameter list isn't well-defined for Vararg tuples, the ith parameter always is.  While I don't want users to need to deal with varargs types, it might make sense to ask what the third parameter of `Tuple{Int, Vararg{String}}` is.  I'm not sure if there's a sensible use-case for that, though, so it may make more sense to start off conservatively and add it if we discover a need.